src/lib/viewers/error/PreviewErrorViewer.js

```diff
@@ -77,8 +76,11 @@ class PreviewErrorViewer extends BaseViewer {
             }
         }
 
+        // Figure out what error message to log and what error message to display
+        const displayMessage = err instanceof Error && err.displayMessage ? err.displayMessage : __('error_default');
```

Comment:
I don't think this logic is completely right. We want messages in this order of preference:

1. err.displayMessage
2. err.message
3. __('error_default')

Maybe:
```
// Up top where const message = reason || __('error_default');
/* eslint-disable no-param-reassign */
err = err instanceof Error ? err : new Error(__('error_default'));
/* eslint-enable no-param-reassign */

// And here
const displayMessage = err.displayMessage || err.message;
```

This helps guard below (`error: err.message`) if err was not properly passed in as an instance of Error as well.

Reply:
I don't think we ever want 2 to be the display message. That could be some stacktrace like `a is undefined`.

For display:
1. err.displayMessage
2. __('error_default')

for logging:
1. err.message
2. err.displayMessage
3.  __('error_defualt')